meta-mender-variscite/recipes-bsp/u-boot/u-boot-variscite-mender-common.inc

```diff
@@ -29,6 +31,27 @@ SRC_URI:remove_var-som-mx6 = " \
     file://0006-env-Kconfig-Add-descriptions-so-environment-options-.patch \
     file://0003-Integration-of-Mender-boot-code-into-U-Boot.patch \
     file://0004-Disable-CONFIG_BOOTCOMMAND-and-enable-CONFIG_MENDER_.patch \
+    file://imx8mm_var_dart_defconfig.mender.patch \
+    file://imx8mm_var_dart.h.mender.patch \
+"
+SRC_URI:remove_imx8mm-var-dart = " \
```

Comment:
That is quite problematic, as a remove operation does not allow for further appends. Yet we should leave that open to the users. So instead of first adding, then removing again, better just add the common things in `SRC_URI`, and then append per board.

Reply:
Additionally, this mixes two separators: ":" and "_". I'm pretty sure that this does not behave as intended.

Reply:
Applied in commit 043c34c531c7e4ad9f3756a4a2dbd18a6f1ff38c